collections/app/lib/Config.scala

```diff
@@ -0,0 +1,30 @@
+package lib
+
+import com.amazonaws.regions.{Regions, Region}
+import com.gu.mediaservice.lib.config.{Properties, CommonPlayAppConfig, CommonPlayAppProperties}
+import com.amazonaws.auth.{BasicAWSCredentials, AWSCredentials}
+
+
+object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
+
+  val appName = "collections"
+
+  val properties = Properties.fromPath("/etc/gu/collections.properties")
+
+  val awsCredentials: AWSCredentials =
+    new BasicAWSCredentials(properties("aws.id"), properties("aws.secret"))
+
+  val dynamoRegion: Region = Region.getRegion(Regions.EU_WEST_1)
+
+  val keyStoreBucket = properties("auth.keystore.bucket")
+
+  val collectionsBucket: String = properties("s3.collections.bucket")
```

Comment:
It does [locally](https://github.com/guardian/grid/blob/master/cloud-formation/dev-template.json#L219-L240) and [publicly](https://github.com/guardian/grid-infra/blob/master/platform/cloud-formation/media-service.json#L609-L630).